composer.json

```diff
@@ -19,7 +19,7 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "php": ">=5.4",
+        "php": ">=7.0",
```

Comment:
Is it necessary?

Reply:
True, I just figured it was safe to bump to 7+ at this point :) Can go back to 5.4 compatible syntax though if you want.

Reply:
Well, there are many projects using Yii 2 with old PHP so I'd release these bugfixes in 2.1.3 version instead of switching to 2.2.0 for requirements change.